snapshot-labs/snapshot
# feat: Add delegation portal

### Issues

Fixes https://github.com/snapshot-labs/snapshot/issues/3893
Fixes https://github.com/snapshot-labs/snapshot/issues/3892

### Changes 

1. Add delegation section to settings
2. Add new space page for delegation portal
3. Enable compound governor delegation standard 

### How to test

1. https://snapshot-m64zxm8pf-snapshot.vercel.app/#/testsnap.eth

OR

1. Go to your space settings -> Delegation
5. Set contract to `0x1f9840a85d5af5bf1d1762f925bdaddc4201f984` and URL to `https://api.thegraph.com/subgraphs/name/arr00/uniswap-governance-v2`
6. Save settings
7. You should see a new 




### Self-review checklist

### Additional notes or considerations

Spaces that have a delegation strategy and delegation setting will see both `Delegate` and `Delegates` in the space sidebar. Snapshot "Delegate" is working exactly the same as before on the UI
